cms/static/js/views/active_video_upload_list.js

```diff
@@ -2,6 +2,7 @@ define(
     ["jquery", "underscore", "backbone", "js/models/active_video_upload", "js/views/baseview", "js/views/active_video_upload", "jquery.fileupload"],
     function($, _, Backbone, ActiveVideoUpload, BaseView, ActiveVideoUploadView) {
         "use strict";
+        var pageUnloadMessage = gettext("Your video uploads are not complete.");
```

Comment:
I found that the browsers tend to add that message themselves (each in their own idiosyncratic ways, of course):
Chrome:
![chrome_unload](https://cloud.githubusercontent.com/assets/7556571/11516033/022e4c46-98d1-11e5-97b6-e861f4f2ab4b.png)
Firefox (shows its own message):
<img width="565" alt="firefox_unload" src="https://cloud.githubusercontent.com/assets/7556571/11516039/06b19f5c-98d1-11e5-803b-a8e3d42a9232.png">
Safari:
<img width="534" alt="safari_unload" src="https://cloud.githubusercontent.com/assets/7556571/11516043/0c66161c-98d1-11e5-82df-660d071f2ec4.png">

But I'm happy to add it if you like.